### PR TITLE
refactor: move resources to testclasses in strings

### PR DESCRIPTION
**Pull request by zielint0**
I need some time to think about it.
Could you guide me, please?

### CODE REVIEW

**Comment by monperrus** [comment]
why is `sourceJar/test.jar` a changed file?

**Comment by zielint0** [comment]
@monperrus 
Because it contains a Java file that was modified (I added .testclasses suffix).
I had to modify Java code (change package) to pass tests.

File: src/test/java/spoon/test/jar/JarTest.java [hunk 1]
```diff
@@ -27,7 +27,7 @@ public void testJar() throws Exception {
 				SpoonResourceHelper.resources("./src/test/resources/sourceJar/test.jar"));
 		Assert.assertTrue(compiler.build());
 		assertEquals(1, factory.getModel().getAllTypes().size());
-		assertEquals("spoon.test.strings.Main", factory.getModel().getAllTypes().iterator().next().getQualifiedName());
+		assertEquals("spoon.test.strings.testclasses.Main", factory.getModel().getAllTypes().iterator().next().getQualifiedName());
```
Comment (surli) [inline src/test/java/spoon/test/jar/JarTest.java]: I don't agree with changing this equals and the related jar file: the purpose of moving the resource file is to ease the maintaining by avoid polluting test classes with resources at the same place. But here the file is already in a jar file, so it does not pollute anything and changing the package name does not change anything to the test.

**Comment by zielint0** [comment]
@surli 
@monperrus 

I am not sure how to explain but will try:
1. create testclasses
2. move Main
3. so I must change package name (add testclasses suffix)
4. fix java code in jar to pass test in JarTest

The result is:
- Main moved to testclasses
- tests pass

What I am doing wrong?
What is incorrect in this try?
Can you move Main using correct way?

**Pull request closed without merging**